workflow/rocoto/gfs_tasks.py

```diff
@@ -665,6 +665,32 @@ def prepoceanobs(self):
 
         return task
 
+    def marineanlletkf(self):
+
+        deps = []
+        dep_dict = {'type': 'metatask', 'name': f'{self.run}_fcst', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
```

Comment:
this should be `enkfgdas`, not `self.run`

Reply:
changed and tested

Reply:
Doesn't look like this change got pushed, it is still `{self.run}`.

Reply:
merged, tested, and pushed